### PR TITLE
feat(FSADT1-1349): Create client for staff Step 1 - Individuals

### PR DESCRIPTION
**Pull request by fterra-encora**
# Description

Adds the front-end side of "Step 1 - Individuals" on create client for staff.

Also fixes the use of undefined/null values on our dropdown component.

Fixes [FSADT1-1349](https://apps.nrs.gov.bc.ca/int/jira/browse/FSADT1-1349)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation update

# How Has This Been Tested?

- [ ] New unit tests
- [ ] New integrated tests
- [ ] New component tests
- [ ] New end-to-end tests
- [ ] New user flow tests
- [ ] No new tests are required
- [ ] Manual tests (description below)
- [ ] Updated existing tests

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
Any successful deployments (not always required) will be available [here](https://nr-forest-client-8-frontend.apps.silver.devops.gov.bc.ca/)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-forest-client/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-forest-client/actions/workflows/merge.yml)

---

Thanks for the PR!

Deployments, as required, will be available below:
Any successful deployments (not always required) will be available [here](https://nr-forest-client-8-frontend.apps.silver.devops.gov.bc.ca/)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/nr-forest-client/actions/workflows/analysis.yml)

After merge, new images are deployed in:
- [Merge Workflow](https://github.com/bcgov/nr-forest-client/actions/workflows/merge.yml)

### CODE REVIEW

File: frontend/src/dto/ApplyClientNumberDto.ts [hunk 1]
```diff
@@ -29,6 +29,10 @@ export interface FormDataDto {
     goodStandingInd: string;
     birthdate: string;
     address: Address;
+    middleName?: string;
```
Comment (paulushcgcj) [inline frontend/src/dto/ApplyClientNumberDto.ts:32]: I would suggest also adding first and last name fields just to make it easier for the backend.

**Review by paulushcgcj — CHANGES_REQUESTED** [review]

File: frontend/src/dto/ApplyClientNumberDto.ts (hunk 1 above)
Comment (fterra-encora) [reply frontend/src/dto/ApplyClientNumberDto.ts:32]: Hey, what I did at the same time is watch the three field names and use them to update the `businessName` field.
So maybe I should actually just remove this `middleName` from the DTO.
What do you think? Or maybe I can keep it and the backend can just ignore it...

Comment (fterra-encora) [reply frontend/src/dto/ApplyClientNumberDto.ts:32]: Ok, I found out the backend actually expects to have the firstName and lastName in the businessInformation as well, so I will do it.

**Comment by fterra-encora** [comment]
The DTO structure was updated according to the new definitions.
And the ID type calculation (to the value expected by the Legacy application) was undone, since the backend is now expecting independent values for ID type and province.

**Review by paulushcgcj — APPROVED** [review]

**Pull request merged**